packages/aws-cdk-lib/aws-docdb/test/cluster.test.ts

```diff
@@ -956,6 +956,138 @@ describe('DatabaseCluster', () => {
       VpcSecurityGroupIds: Match.arrayWith([stack.resolve(securityGroup.securityGroupId)]),
     });
   });
+
+  test('can create a cluster with snapshot removal policy', () => {
+    // GIVEN
+    const stack = testStack();
+    const vpc = new ec2.Vpc(stack, 'VPC');
+
+    // WHEN
+    new DatabaseCluster(stack, 'Database', {
+      instances: 1,
+      masterUser: {
+        username: 'admin',
+        password: cdk.SecretValue.unsafePlainText('tooshort'),
+      },
+      instanceType: ec2.InstanceType.of(ec2.InstanceClass.BURSTABLE2, ec2.InstanceSize.SMALL),
+      vpc,
+      removalPolicy: cdk.RemovalPolicy.SNAPSHOT,
+    });
+
+    // THEN
+    Template.fromStack(stack).hasResource('AWS::DocDB::DBCluster', {
+      Properties: {
+        DBSubnetGroupName: { Ref: 'DatabaseSubnets56F17B9A' },
+        MasterUsername: 'admin',
+        MasterUserPassword: 'tooshort',
+        VpcSecurityGroupIds: [{ 'Fn::GetAtt': ['DatabaseSecurityGroup5C91FDCB', 'GroupId'] }],
+      },
+      DeletionPolicy: 'Snapshot',
+      UpdateReplacePolicy: 'Snapshot',
+    });
+
+    // Associated instance gets delete policy
+    Template.fromStack(stack).hasResource('AWS::DocDB::DBInstance', {
+      DeletionPolicy: 'Delete',
+      UpdateReplacePolicy: 'Delete',
+    });
+
+    // Associated security group gets delete policy
+    Template.fromStack(stack).hasResource('AWS::EC2::SecurityGroup', {
+      DeletionPolicy: 'Delete',
+      UpdateReplacePolicy: 'Delete',
+    });
+  });
+
+  test('can specify instances removal policy', () => {
+    // GIVEN
+    const stack = testStack();
+    const vpc = new ec2.Vpc(stack, 'VPC');
+
+    // WHEN
+    new DatabaseCluster(stack, 'Database', {
+      instances: 1,
+      masterUser: {
+        username: 'admin',
+        password: cdk.SecretValue.unsafePlainText('tooshort'),
+      },
+      instanceType: ec2.InstanceType.of(ec2.InstanceClass.BURSTABLE2, ec2.InstanceSize.SMALL),
+      vpc,
+      instanceRemovalPolicy: cdk.RemovalPolicy.DESTROY,
```

Comment:
Can we have this test instead as `SNAPSHOT` removal policy with instance removal policy `RETAIN`?